src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/properties/Locale.xml

```diff
@@ -17,4 +17,5 @@
 >
     <property resource="Sylius\Component\Locale\Model\Locale" name="id" identifier="false" writable="false" />
     <property resource="Sylius\Component\Locale\Model\Locale" name="code" identifier="true" required="true" />
+    <property resource="Sylius\Component\Locale\Model\Locale" name="name" writable="false" />
```

Comment:
What about `createdAt` and `updatedAt` properties, to stay consistent?

Reply:
I would like not to change the current responses during upgrading to APIP 3 🖖🏻